internal/distro/rhel85/partition_tables.go

```diff
@@ -1,244 +1,41 @@
 package rhel85
 
 import (
-	"io"
-	"math/rand"
-
-	"github.com/google/uuid"
-	"github.com/osbuild/osbuild-composer/internal/blueprint"
 	"github.com/osbuild/osbuild-composer/internal/disk"
 	"github.com/osbuild/osbuild-composer/internal/distro"
 )
 
-const (
-	sectorSize = 512
-
-	biosBootPartitionType = "21686148-6449-6E6F-744E-656564454649"
-	biosBootPartitionUUID = "FAC7F1FB-3E8D-4137-A512-961DE09A5549"
-
-	bootPartitionType = "0FC63DAF-8483-4772-8E79-3D69D8477DE4"
-	bootPartitionUUID = "CB07C243-BC44-4717-853E-28852021225B"
-
-	bootEFIPartitionType  = "C12A7328-F81F-11D2-BA4B-00A0C93EC93B"
-	bootEFIPartitionUUID  = "68B2905B-DF3E-4FB3-80FA-49D1E773AA33"
-	bootEFIFilesystemUUID = "7B77-95E7"
-
-	defaultPartitionType = "0FC63DAF-8483-4772-8E79-3D69D8477DE4"
-	rootPartitionUUID    = "6264D520-3FB9-423F-8AB8-7A0A8E3D3562"
-)
+var defaultArches = map[string]disk.PartitionTable{
+	distro.X86_64ArchName:  gptPartitionTable,
+	distro.Aarch64ArchName: gptPartitionTable,
+	distro.Ppc64leArchName: dosPartitionTable,
+	distro.S390xArchName:   dosPartitionTable,
+}
 
-var x86PartitionTable = disk.PartitionTable{
-	UUID:       "D209C89E-EA5E-4FBD-B161-B461CCE297E0",
-	Type:       "gpt",
-	Partitions: []disk.Partition{},
+var ec2ValidArches = map[string]disk.PartitionTable{
+	distro.X86_64ArchName:  gptPartitionTable,
+	distro.Aarch64ArchName: gptPartitionTable,
```

Comment:
So my idea when previously making the suggestion was that `defaultArches` and `ec2ValidArches` would hold the default partitioning scheme, probably very similarly defined as it is done currently https://github.com/osbuild/osbuild-composer/blob/1e2ba4da64190b3ada6d651192ac1bb4d53cd60c/internal/distro/rhel85/partition_tables.go#L12-L145

Then the `disk.CreatePartitionTable()` would basically just combine it with the provided Blueprint customisations and produce a potentially different version of the provided partition table on the output.

My idea was also that we could potentially omit some values in the definition of default / base partitioning table and le the `disk.CreatePartitionTable()` finalise the partitioning table before it is usable. This could be the start offset (it should be possible to calculate it based on the provided list of partitions and their sizes). and UUIDs. This is however not really that important.

PS: I would also suggest to make the names of these variables more self-explanatory and based more on their usage (default per-arch partitioning scheme).